Refactored: Component Library

ShekharDhangar opened the pull request:
Refactored: the component Library

ShekharDhangar committed:
Refactored: Component Library

Refactored: the component Library

MayankKumar10 commented on components/Badge/badge.css:
```diff
@@ -55,10 +55,11 @@
     position: absolute;
-    font-size: 1.75rem;
+    font-size: 1.5rem;
     top: 0rem;
```
use top 0 and left 0 in HTML tag so you don't have to write it will inherit by its child components. 
And you don't have to use rem when you use 0.

ShekharDhangar replied:
Actually then the top:0 will reflect everywhere right, in someplaces I am using top 1rem,2rem, so then anyway I have to write top:1rem or 2rem,right?

MayankKumar10 replied:
The child class values override the values of the parent class, i.e when you define globally it will inherit from its parent but if you want to change it you can, by giving different values in child class or id.

MayankKumar10 reviewed (approved):
Everything else Looks good to me you can merge it

abhisheksahani-nova reviewed (approved):
Make changes said by mayank, else looks good to merge

ShekharDhangar committed:
Update: Readme

Updated Readme File and shifted buttons style of input from input file to button  component file

b0llu commented on components/Badge/badge.css:
```diff
@@ -4,28 +4,28 @@
     position: absolute;
     border-radius: 50%;
-    border: 0.5rem solid white;
+    border: 0.5rem solid var(--clr-white);
```
you can use px if the value is less than 1 rem

b0llu commented on components/Badge/badge.css:
```diff
@@ -4,28 +4,28 @@
 .badge-sm{
-    border: 0.2rem solid white;
+    border: 0.2rem solid var(--clr-white);
     right: 0.2rem;
```
here also you can use px

b0llu commented on components/Badge/badge.css:
```diff
@@ -4,28 +4,28 @@
 }
 .badge-lg{
-    border: 0.4rem solid white;
+    border: 0.4rem solid var(--clr-white);
```
rem to px

ShekharDhangar replied:
Done

b0llu commented on README.md:
```diff
@@ -1,2 +1,52 @@
+
+
+### Things I want to improve/explore further
+- Component library in React
```
Readme is looking good

b0llu reviewed (approved):
Suggested few changes! The refactoring is looking good.

ShekharDhangar committed:
Changed less than 1rem units to px

Changed less than 1rem units to px
